app/models/concerns/audit_search.rb

```diff
@@ -4,15 +4,16 @@ module AuditSearch
   included do
     belongs_to :user, :class_name => 'User'
     belongs_to :search_users, :class_name => 'User', :foreign_key => :user_id
-    belongs_to :search_hosts, -> { where(:audits => { :auditable_type => 'Host::Base' }) },
+    belongs_to :search_hosts, -> { joins('INNER JOIN audits ON audits.auditable_id = hosts.id AND audits.auditable_type LIKE \'Host::%\'') },
```

Comment:
The scope (and the one for nics) was probably never evaluated before. The auditable type for hosts should be `Host::Base`, but `Host::Base` is not `auditable` so I had to hack around a bit

Reply:
What if we go with a workaround, which seems to make that association work and return the valid results? Something like:
```ruby
    belongs_to :search_hosts, :class_name => 'Host::Base', :foreign_key => :auditable_id

    def search_hosts
      return nil unless auditable_type&.match?(/^(::)?Host::/)
      super
    end
```